.gitignore

```diff
@@ -33,6 +33,9 @@ test.py
 do_release.py
 *.env*
 
+# mk docs build
+site/
+

```

Comment:
**question:** why is this needed since we removed mk docs?

Reply:
Just missed it when in my search to remove, it's not needed